gcc/rust/backend/rust-compile-base.cc

```diff
@@ -695,6 +701,8 @@ HIRCompileBase::compile_constant_item (
 
       gcc_assert (TREE_CODE (bind_tree) == BIND_EXPR);
       DECL_SAVED_TREE (fndecl) = bind_tree;
+      DECL_DECLARED_CONSTEXPR_P (fndecl);
```

Comment:
Shouldn't this be
```suggestion
      DECL_DECLARED_CONSTEXPR_P (fndecl) = 1;
```
?

Reply:
Fixed.